### PR TITLE
Add to version checked APIs guidance

### PR DESCRIPTION
**Pull request by aeunms**
Adds detail about adding an isSupported() function for version checked APIs addition guidance. Also fixes broken link in CONTRIBUTING.md.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -64,7 +73,7 @@ Exa)
 
 If you're adding a capability onto an already existing version requirement, simply add your object into the existing array.
```
Comment (erinha) [inline CONTRIBUTING.md:74]: nit: "If you're adding a capability **to** an already existing version requirement, simple add your object **to** the existing array."

(Bold to emphasize which words should be edited)

**Review by erinha — CHANGES_REQUESTED** [review]
Thanks for making these fixes, it looks much better! Small comment since you're here fixing this file already. :)

File: CONTRIBUTING.md (hunk 1 above)
Comment (aeunms) [reply CONTRIBUTING.md]: Sounds good, changes pushed!

**Review by erinha — DISMISSED** [review]
🚀

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -40,12 +40,26 @@ the PR.
 
 ### Adding an API that utilizes version checks for compatibility
 
+This option should only be used for work that meets ALL of the below requirements:
+
+- Work which has already been discussed with the TeamsJS devs as using this approach
```
Comment (erinha) [inline CONTRIBUTING.md:45]: This is a great addition! Suggestion for some rewording that I think adds clarity:

- Features which have already been discussed with the TeamsJS owners and for which approval to use this approach has been granted,
- Feature implementation that has a requirement of running in host clients that have not onboarded to the new declarative capability support architecture.

Note: typo in existing text ("capabiity")

**Review by erinha — CHANGES_REQUESTED** [review]
Thanks for adding the text we discussed offline. A small suggestion for rewording, but great overall!

File: CONTRIBUTING.md (hunk 2 above)
Comment (aeunms) [reply CONTRIBUTING.md:45]: Great suggestions. Will make!

**Review by erinha — APPROVED** [review]

**Review by shrshindeMSFT — APPROVED** [review]

**Pull request merged**